apps/examples/testcase/le_tc/network/itc_net_inet.c

```diff
@@ -0,0 +1,239 @@
+/****************************************************************************
+*
+* Copyright 2017 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+****************************************************************************/
+
+/// @file itc_net_inet.c
+/// @brief ITC Test Case Example for inet() API
+
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+#include "tc_internal.h"
+
+/**
+* @testcase          :itc_net_inet_addr_n
+* @brief             :function converts the Internet host address cp from IPv4
+*                     numbers-and-dots notation into binary data in network byte order.
+* @scenario          :passing invalid address
+* @apicovered        :inet_addr()
+* @precondition      :None
+* @postcondition     :None
+*/
+static void itc_net_inet_addr_n(void)
+{
+	int ret;
+
+	ret = inet_addr("q");
+	TC_ASSERT_EQ("inet", ret, -1);
+	if (ret != -1)
```

Comment:
Unnecessary codes. If ret is not -1, TC_ASSERT macro returns.

Reply:
changes are done as per your suggestion. please review